pkg/tests/tests.go

```diff
@@ -0,0 +1,58 @@
+package tests
+
+import (
+	"net/http"
+
+	"github.com/cloud-bulldozer/ocm-api-load/pkg/helpers"
+	"github.com/cloud-bulldozer/ocm-api-load/pkg/tests/handlers"
+)
+
+// Specify Test Cases
+// They are written this way to re-use functionality where possible and
+// hopefully make it easier to modify and/or extend given the declarative
+// style.
+var tests = []helpers.TestOptions{
+	{
+		TestName: "self-access-token",
+		Path:     "/api/accounts_mgmt/v1/access_token",
+		Method:   http.MethodPost,
+		Handler:  handlers.TestStaticEndpoint,
+	},
+	{
+		TestName: "list-subscriptions",
+		Path:     "/api/accounts_mgmt/v1/subscriptions",
+		Method:   http.MethodGet,
+		Handler:  handlers.TestStaticEndpoint,
+	},
+	{
+		TestName: "access-review",
+		Path:     "/api/authorizations/v1/access_review",
+		Method:   http.MethodPost,
+		Body:     "{\"account_username\": \"rhn-support-tiwillia\", \"action\": \"get\", \"resource_type\": \"Subscription\"}",
```

Comment:
This can be created using the ocm SDK:
```
import (
   authv1  "https://github.com/openshift-online/ocm-sdk-go/authorizations/v1"
)
...

resourceReviewReq, err :=  authv1.NewAccessReviewRequest().AccountUsername("rhn-support-tiwillia").Actions("get").ResourceType("Subscription").Build
if err != nil {
  // do some stuff...
}
```

I know its a hassle and it might require you using an ad-hoc handler but I prefer we use the typed sdk instead of byte buffers.

Reply:
Agreed. This is actually just copied + pasted from a prior commit that I made. Can we address this in a separate issue? I can open one if so.

Reply:
Sure.

Reply:
Opened a new issue with this.